src/services/tossPaymentsService.ts

```
@@ -0,0 +1,51 @@
+    }
+    return `Basic ${Buffer.from(secretKey + ':').toString('base64')}`;
+  }
+
+  // 결제 상태 확인
+  static async getPaymentStatus(paymentKey: string) {
+    const response = await fetch(`https://api.tosspayments.com/v1/payments/${paymentKey}`, {
+      method: 'GET',
+      headers: {
+        Authorization: this.getAuthHeader(),
+        'Content-Type': 'application/json',
+      },
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)